### PR TITLE
New blogpost: AMY release v1.5.1

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**

### CODE REVIEW

File: _posts/2016/04/2016-04-03-AMY-release-151.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+time: "11:00:00"
+category: ["AMY", "Google Summer of Code", "Tooling"]
+---
+
+We're very pleased to announce the newest ([v1.5.1][]) release of [AMY][], our
+workshop management tool.  This release is special because:
+
+1. it contains more bugfixes than new features
+2. we had a number of submissions from prospect
+   [Google Summer of Code 2016](https://summerofcode.withgoogle.com/) students.
```
Comment (rgaiacs) [inline _posts/2016/04/2016-04-03-AMY-release-151.md]: "students" -> "(GSoC) students"

File: _posts/2016/04/2016-04-03-AMY-release-151.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+GSOC students that submitted fixes or new features to AMY:
```
Comment (rgaiacs) [inline _posts/2016/04/2016-04-03-AMY-release-151.md]: GSoC

File: _posts/2016/04/2016-04-03-AMY-release-151.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+* [Shubham Singh][] added "Notes" field to instructor profile update form
+* [Nikhil Verma][] found and fixed "List duplicates" page error when no
+  duplicates existed
+* [Shubham Singh][] added new tag "hackaton"
+* [Chris Medrela][] found and fixed 404 page for revisions that didn't exist
+
+Additionally [Nikhil Verma][] is working on getting certificates to generate
+from within [AMY][], but this will probably land in [v1.5.2][].
+
+Apart from GSOC students we've had a number of contributions from our very
```
Comment (rgaiacs) [inline _posts/2016/04/2016-04-03-AMY-release-151.md]: GSoC

**Comment by rgaiacs** [comment]
Looks good to me. @pbanaszkiewicz Thanks. :sparkles:

File: _posts/2016/04/2016-04-03-AMY-release-151.md (hunk 1 above)
Comment (pbanaszkiewicz) [inline _posts/2016/04/2016-04-03-AMY-release-151.md]: Do you want me to change "prospect [Google Summer of Code 2016](https://summerofcode.withgoogle.com/) students" to "prospect (GSoC) students"?

Comment (gvwilson) [inline _posts/2016/04/2016-04-03-AMY-release-151.md]: "prospective", not "prospect".

Comment (rgaiacs) [inline _posts/2016/04/2016-04-03-AMY-release-151.md]: > from prospect [Google Summer of Code 2016](https://summerofcode.withgoogle.com/) (GSoC) students.

Just to make clear what GSoC means.

**Comment by pbanaszkiewicz** [comment]
@gvwilson, @rgaiacs fixed! Merge away :-)

**Comment by jduckles** [comment]
Thanks @pbanaszkiewicz! Awesome work.

**Pull request merged**